com.unity.visualeffectgraph/Editor/GraphView/VFXViewWindow.cs

```diff
@@ -274,6 +274,13 @@ void Update()
                             graph.RecompileIfNeeded(true, true);
 
                         controller.RecompileExpressionGraphIfNeeded();
+
+                        // Hack to avoid infinite recompilation (due to subgraphs implementation...) TODO: Fix correctly
+                        if (graph.IsExpressionGraphDirty())
+                        {
+                            Debug.LogError("Expression graph was marked as dirty after compiling context for UI. Discard to avoid infinite compilation loop.");
```

Comment:
This messages is causing a failure in editor Performance Test :
```[08:01:20.332 INF] [112/139] TestFinish
	name: UnityEditor.VFX.PerformanceTest.VFXCompilePerformanceTests.VFXViewWindow_Open_And_Render("SimpleMeshOutput")
	result: FAILED
	message: Unhandled log message: '[Error] Expression graph was marked as dirty after compiling conte
[08:01:20.339 INF] xt for UI. Discard to avoid infinite compilation loop.'. Use UnityEngine.TestTools.LogAssert.Expect
	stackTrace: UnityEditor.EditorApplication:Internal_CallUpdateFunctions ()```

See [here](https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/vfx%2Ffix%2F1298466/.yamato%2Fvfx_urp-win-dx11.yml%23VFX_URP_Win_DX11_editmode_PerformanceTests_mono_Linear_trunk/5356463/job)

Maybe it's an instability, the test has been run only once.

We can either : 
- Disable logging while running any performanceTest
- Replace LogError by LogMessage ? (I'm not sure it's enough)
- Add a condition on this LogError